src/components/dropdown/Dropdown.tsx

```diff
@@ -1,10 +1,12 @@
+import * as classNames from 'classnames';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 
 export interface IDropdownOwnProps extends React.ClassAttributes<Dropdown> {
     id?: string;
     toggleContent: JSX.Element[];
     dropdownItems: JSX.Element[];
+    className?: string;

```

Comment:
I think you don't need to define it here since `IDropdownOwnProps extends React.ClassAttributes<Dropdown>`